### ISSUE TITLE
not authorized to send Apple events to finder 

### ISSUE THREAD

**Issue by msmithee** [apprenticeharper/DeDRM_tools#698]
I tried opening 6.5.4 and got this message:

[Screen Shot.pdf](https://github.com/apprenticeharper/DeDRM_tools/files/2659307/Screen.Shot.pdf)

**Comment by ElleKayEm** [comment]
Have you tried with the current version 6.6.1?  Also, take a look at this section of the FAQs:

https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#the-macintosh-application

**Comment by msmithee** [comment]
Yes I have and got the "The application "DeDRM" can't be opened". 
I looked at the issue #505, but don't know how to make that work.

**Comment by ElleKayEm** [comment]
Entering the command in the terminal?  As suggested here:

https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#i-cant-open-the-macintosh-application-at-all-i-get-the-aplication-dedrm-cant-be-opened
